coalib/settings/FunctionMetadata.py

```diff
@@ -13,12 +13,13 @@ class FunctionMetadata:
     @enforce_signature
     def __init__(self,
                  name: str,
-                 desc: str='',
-                 retval_desc: str='',
-                 non_optional_params: (dict, None)=None,
-                 optional_params: (dict, None)=None,
-                 omit: (set, tuple, list, frozenset)=frozenset(),
-                 deprecated_params: (set, tuple, list, frozenset)=frozenset()):
+                 desc: str = '',
+                 retval_desc: str = '',
+                 non_optional_params: (dict, None) = None,
+                 optional_params: (dict, None) = None,
+                 omit: (set, tuple, list, frozenset) = frozenset(),
+                 deprecated_params:
+                     (set, tuple, list, frozenset) = frozenset()):
```

Comment:
this is too weird.
drop the trailing `):` to the next line, unindented.
(look for similar cases)
(im still reviewing, so dont push yet)